utils/luci-app-owm/luasrc/owm.lua

```diff
@@ -205,6 +205,15 @@ function showmac(mac)
 	return mac
 end
 
+function get_position()
+	local position = {}
+	uci:foreach("system", "system", function(s) --owm
+		position['latitude'] = tonumber(s.latitude) --owm
```

Comment:
cleaned / moved them back to original positions. but there are many of these "--owm" comments.

looks like they are marking the datafields of the OWM-json
@stargieg what do you think